packages/artillery-plugin-expect/index.js

```diff
@@ -83,10 +83,13 @@ function expectationsPluginOnError(
   events,
   done
 ) {
-  if (userContext.expectationsPlugin.outputFormat === 'json') {
-    artillery.log(JSON.stringify({ ok: false, error: scenarioErr.message }));
+  if (scenarioErr.message.startsWith('Failed expectations for request')) {
```

Comment:
This will work for now but probably better to handle this without relying on the error message, e.g. with a custom error class.